templates/metal/private/metal_private_hca0.h.j2

```diff
@@ -0,0 +1,17 @@
+#ifndef __METAL_DT_HCA0__H
```

Comment:
If we're keeping this default driver support for the HCA, we should probably remove the '0' version number from the "HCA0" in the name of this file and associated content.

Reply:
version hca1 will be coming with additional functionality.